apps/files_sharing/src/views/SharingTab.vue

```diff
@@ -230,9 +230,9 @@ export default {
 			shareDetailsData: {},
 			returnFocusElement: null,
 
-			internalSharesHelpText: t('files_sharing', 'Use this method to share files with individuals or teams within your organization. If the recipient already has access to the share but cannot locate it, you can send them the internal share link for easy access.'),
-			externalSharesHelpText: t('files_sharing', 'Use this method to share files with individuals or organizations outside your organization. Files and folders can be shared via public share links and email addresses. You can also share to other Nextcloud accounts hosted on different instances using their federated cloud ID.'),
-			additionalSharesHelpText: t('files_sharing', 'Shares that are not part of the internal or external shares. This can be shares from apps or other sources.'),
+			internalSharesHelpText: t('files_sharing', 'Share files within your organization. Recipients who can already view the file can also use this link for easy access.'),
+			externalSharesHelpText: t('files_sharing', 'Share files with others outside your organization via public links and email addresses. You can also share to Nextcloud accounts on other instances using their federated cloud ID.'),
+			additionalSharesHelpText: t('files_sharing', 'Shares from apps or other sources which are not internal or external.'),
```

Comment:
I think the last part is a bit confusing, if its only me then you should add a comment for translators as there would be likely also confused ones :sweat_smile:
At least for me its easier readable if

```suggestion
			additionalSharesHelpText: t('files_sharing', 'Shares from apps or other sources which are not included in internal or external shares.'),
```

To make clear that the last part is about the two other sections in the UI and not about the `sources`.